test/recipes/75-test_quicapi_data/ssltraceref.txt

```diff
@@ -297,4 +297,6 @@ Sent Packet
   Payload length: 60
   Packet Number: 0x00000000
 Sent Datagram
-  Length: 1200
+  Length: 1119
+Sent Datagram
+  Length: 81
```

Comment:
It's slightly surprising that this PR would impact the ssltraceref.txt. I'm not really sure why it has.